api/api-doc/storage_service.json

```diff
@@ -1057,7 +1057,16 @@
                "produces":[
                   "application/json"
                ],
-               "parameters":[]
+               "parameters":[
+                    {
+                     "name":"global",
+                     "description":"true if cleanup of entire cluster is requested",
+                     "required":true,
+                     "allowMultiple":false,
+                     "type":"boolean",
+                     "paramType":"path"
```

Comment:
Do I understand correctly that you chose a path parameter over a query parameter because you want to make it required?